### PR TITLE
N/M variant of HBC

### PR DESCRIPTION
**Pull request by Anush2303**
## New Behavior

Added N/M variant of Multi Stacked Horizontal bar chart.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts [hunk 1]
```diff
@@ -13,6 +13,12 @@ export default {
 
 export const Basic: Story<FluentHorizontalBarChart> = renderComponent(storyTemplate).bind({});
 
+export const NMVariant: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
+  <div>
+    <fluent-horizontalbarchart variant="part-to-whole"> </fluent-horizontalbarchart>
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:18]: use this as an enum similar to absolute variant

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.ts [hunk 2]
```diff
@@ -428,11 +431,23 @@ export class HorizontalBarChart extends FASTElement {
...
       .attr('class', 'chartTitle')
       .text(data?.chartTitle ? data?.chartTitle : '');
 
+    const hideNumber = this.hideRatio === undefined ? false : this.hideRatio;
+
+    const showRatio = this.variant !== Variant.AbsoluteScale && !hideNumber && data!.chartData!.length === 2;
+    const getChartData = () => (data!.chartData![0].data ? data!.chartData![0].data : 0);
+
+    if (showRatio) {
+      const ratioDiv = chartTitleDiv.append('div').attr('role', 'text');
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:447]: Where is the data for the image that you shared.
image

Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:447]: Line 448,449

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts (hunk 1 above)
Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts:18]: Enum is already defined. This is passed as an attribute similar to prop that we used in React.

**Pull request closed without merging**